interfaces/cython/cantera/base.pyx

```diff
@@ -111,8 +109,21 @@ cdef class _SolutionBase:
         elif source:
             root = AnyMapFromYamlString(stringify(source))
 
-        phaseNode = root[stringify("phases")].getMapWhere(stringify("name"),
-                                                          stringify(name))
+        if name == 'gri30_mix' or name == 'gri30_multi':
+            # catch transitional behavior
+            warnings.warn(
+                "The phases 'gri30_mix' and 'gri30_multi' are no longer "
+                "defined in the input file 'gri30.yaml'. The YAML version uses "
+                "the 'Mix' transport manager by default instead, which can be "
+                "overridden by specifying a transport manager via the "
+                "`transport_model' keyword argument during initialization. "
+                "Support for transitional behavior will be removed after "
+                "Cantera 2.5.", DeprecationWarning)
```

Comment:
I think the message is a little too broad, and I'd prefer if it was explicit about how users could fix the warning. Something like:
```
The phases named 'gri30_mix' and 'gri30_multi' are no longer defined
in the input file 'gri30.yaml'. By default, 'gri30.yaml' uses the mixture-averaged
transport model. The following options are available:

* ct.Solution('gri30.yaml')  # Mixture-averaged transport
* ct.Solution('gri30.yaml', transport_model="mix")  # Mixture-averaged transport
* ct.Solution('gri30.yaml', transport_model=None)  # No transport model, useful for 0-D simulations
* ct.Solution('gri30.yaml', transport_model="multicomponent")  # Multicomponent transport
```
This is now bordering on detailed documentation... maybe some of the examples could be eliminated in favor of a link to a page on our website (or on the GH Wiki) with a more detailed explanation. Regardless, I think at least one example of the appropriate syntax should be kept.